charts/clusterapi-resources/templates/ClusterResourceSet.yaml

```
@@ -33,3 +34,7 @@ spec:
   - kind: Secret
     name: {{ $s.name }}
 {{- end }}
+{{- if .Values.argo.enabled }}
+  - kind: ConfigMap
+    name: add-cluster-to-argo
```

Forgot to add the new secret with argo token
```suggestion
    name: add-cluster-to-argo
 - kind: Secret
    name: add-cluster-to-argo
```